src/sentry/auth/providers/github/client.py

```diff
@@ -21,15 +21,12 @@ def __init__(self, client_id, client_secret):
         self.http = http.build_session()
 
     def _request(self, path, access_token):
-        params = {"client_id": self.client_id, "client_secret": self.client_secret}
-
         headers = {"Authorization": "token {0}".format(access_token)}
+        auth = (self.client_id, self.client_secret)
```

Comment:
@slohmes I would be surprised if we actually need to provide the client_id and client secret if we already have an `Authorization` header. Does the auth still work if you remove those parameters?

Reply:
It does, thanks!